ParisBNF/abb/BNFabb51.xml

```diff
@@ -0,0 +1,495 @@
+                              <ab type="ruling"/>
+                              <ab type="pricking">Pricking and ruling hardly visible.</ab>
+                              <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C.</ab>
+                              <ab type="ruling">The upper line is written above the ruling.</ab>
+                              <ab type="ruling">The bottom line is written above the ruling.</ab>
+                              </layout>
+
+                              <layout columns="2" writtenLines="26 29" xml:id="textarea2">
+                                <locus from="78ra" to="130vb"/>
+                                <ab type="ruling"/>
+                                <ab type="pricking">Pricking and ruling hardly visible.</ab>
+                                <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C.</ab>
+                                <ab type="ruling">The upper line is written above the ruling.</ab>
+                                <ab type="ruling">The bottom line is written above the ruling.</ab>
+                              </layout>
+
+
+                              <layout columns="2" writtenLines="27 28" xml:id="textarea3">
+                                <locus from="131ra" to="157vb"/>
+                              <ab type="ruling"/>
+                              <ab type="pricking">Pricking and ruling hardly visible.</ab>
+                              <ab type="ruling" subtype="pattern">1A-1A-1A1A/0-0/0-0/C.</ab>
+                              <ab type="ruling">The upper line is written above the ruling.</ab>
+                              <ab type="ruling">The bottom line is written above the ruling.</ab>
+                            </layout>
+                           </layoutDesc>
+                          </objectDesc>
+                        <handDesc>
+                           <handNote xml:id="h1" script="Ethiopic">
+                             <locus from="1ra" to="77vb"/>
+                              <seg type="script">Late 15th/first half of the 16th century.</seg>
+                              <seg type="ink">Black; red.</seg>
+                              <desc>Careful, very irregular; by a mediocre scribe.
+                                Right-sloping, broadly spaced letters. The slant and size of the letters deviate significantly;
+                                the written lines are
+                                biased on many folia. The word and sentence dividers are set up very close to
+                              the bottom line (under the vowel marker if the preceding characther has any);
+                              especially the word dividers are compressed,
+                              occupy very small space.</desc>
+                              </handNote>
+
+                              <handNote xml:id="h2" script="Ethiopic">
+                                <locus from="78ra" to="130vb"/>
```

Comment:
can it be that there is a change in msPart? new text, new quire, new layout, new hand?

Reply:
It may look like, but the combination of evidence shows that the ms is unlikely composite. Scholars speak of "codicological units", but stop short of speaking of production units. It is a complex ms, though, the images are not so good.